src/content/docs/kubernetes-pixie/kubernetes-integration/installation/upgrading-kubernetes.mdx

```diff
@@ -0,0 +1,72 @@
+
+If you want to upgrade manually your Kuberenetes instegration installed via Helm:
+
+1. Update the local chart repository:
+
+   ```shell
+   helm repo update
+   ```
+
+2. Update the release by running again the appropriate `helm upgrade --install ...` command in the section above:
+
+   ```shell
+   helm upgrade --install newrelic newrelic/nri-bundle \
+   --namespace newrelic --create-namespace \
+   -f values-newrelic.yaml
+   ```
+
+### Upgrading using the guided install and plain manifests [#update-manifests]
+
+<Callout variant="tip">
+   We encourage you to deploy our integration using Helm, as it provides a cleaner upgrade path compared to using manifests directly.
+</Callout>
+
+If custom manifests have been used instead of Helm, you will need to first remove the old installation using `kubectl delete -f previous-manifest-file.yml`, and then proceed through the guided install again. This will generate an updated set of manifests that can be deployed using `kubectl apply -f manifest-file.yml`.
+
+We don't recommend applying a new version of the manifest file without removing the previous one first, since it might leave some leftover components in your cluster.
+
+If you need an advanced way to create your manifest, template it using Helm.
+
+## Upgrading to the new version of KSM [#update-ksm]
+
+### Changed label selectors [#changed-label-selectors]
+
+When upgrading to the new version of KSM, you might see an error like this:
+```shell
+   v1.LabelSelector{MatchLabels:map[string]string{"app.kubernetes.io/instance":"newrelic", "app.kubernetes.io/name":"kube-state-metrics"}, MatchExpressions:[]v1.LabelSelectorRequirement(nil)}: field is immutable
+   ```
+The label selector in the new version of the chart changed, but the name of the deployment did not. Therefore, in order to upgrade you need to remove the kube-state-metrics deployment, change the release name or re-create the release.
+
+### Deprecated values [#deprecated-values]
+On the other hand, you can see the following log when upgrading:
+```shell
+   warning: skipped value for nri-bundle.kube-state-metrics.collectors: Not a table.
+Error: UPGRADE FAILED: template: nri-bundle/charts/kube-state-metrics/templates/role.yaml:18:6: executing "nri-bundle/charts/kube-state-metrics/templates/role.yaml" at <has "certificatesigningrequests" $.Values.collectors>: error calling has: Cannot find has on type map
+   ```
+This is because of the KSM chart values changed and the map of collectors has been changed to a list of collectors.
+The collector list is not needed anymore, and it has been removed from the new values.
+
+If you still have those in your static Helm values, or if you are using the flag --reuse-values you should first remove the following values:
+```
+collectors:
+    certificatesigningrequests: false
+    ingresses: false
+```
```

Comment:
I again think this requires more context on where this key is located within `values.yaml`.

Reply:
I don't have context on this, can @roobre clarify?

Reply:
I've replaced it by this:
kube-state-metrics:
  collectors:
    certificatesigningrequests: true
    ingresses: false